src/applications/personalization/profile-2/components/ProfileMobileSubNav.jsx

```diff
@@ -178,7 +180,6 @@ const ProfileMobileSubNav = ({
               ref={openMenuButton}
               className="open-menu"
               type="button"
-              aria-controls="va-profile-sidebar"
```

Comment:
now that the mobile nav is an all-in-one component I don't think this attribute is needed at all.